src/utils/daos.c

```diff
@@ -45,15 +45,25 @@
 #include <daos/rpc.h>
 #include <daos/debug.h>
 #include <daos/object.h>
+#include <sys/stat.h>
 
 #include "daos_types.h"
 #include "daos_api.h"
 #include "daos_uns.h"
+#include "daos_fs.h"
 #include "daos_hdlr.h"
 #include "dfuse_ioctl.h"
 
 const char		*default_sysname = DAOS_DEFAULT_SYS_NAME;
 
+static enum fs_op
+filesystem_op_parse(const char *str)
+{
+	if (strcmp(str, "copy") == 0)
+		return FS_COPY;
+	return 0;
```

Comment:
I think this is supposed to return -1 for errors.
```suggestion
	return -1;
```